packages/app/src/app/overmind/namespaces/dashboard/actions.ts

```diff
@@ -398,6 +400,75 @@ export const getSandboxesByPath: AsyncAction<string> = async (
   }
 };
 
+export const getReposByPath: AsyncAction | Action<string> = async (
```

Comment:
```suggestion
export const getReposByPath: AsyncAction<string> = async (
```

Reply:
because of line 421 It can also be a normal actions

Reply:
nvm, done